src/utils/vaultMath.ts

```diff
@@ -119,6 +122,9 @@ export const lockedAfterChange = (
   return AmountMath.makeEmpty(locked.brand);
 };
 
+const lowestPrice = (priceA: Ratio, priceB?: Ratio) =>
+  priceB ? (ratioGTE(priceA, priceB) ? priceB : priceA) : priceA;
```

Comment:
this would be handy in `ratio.js`

Reply:
In zoe? I guess, it's small enough and only used in the UI though.

Reply:
Not trying to make more work :-) 

It illustrates the absence of max/min in ratio math. It's simple but also something one can expect to be provided by a math lib. This came up for TimeMath recently.

Reply:
Ah I see, yea I wasn't sure if this would be useful more generally with the optional `priceB?`